### PR TITLE
Copy TAP files to testAQApipeline jenkinsjob

### LINKED ISSUE #3678: archive tap/metadata files to pipeline parent jenkins job for marketplace
Currently gathering TAP/metadata files for marketplace without artifactory service has to be done manually. Supporting archive tap/metadata files to pipeline parent jenkins job would greatly smooth the process.

#3640

### PR DESCRIPTION
**Pull request by sophia-guo**
Fix #3678 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
https://ci.adoptopenjdk.net/view/Test_grinder/job/AQA_Test_Pipeline/76/

File: buildenv/jenkins/aqaTestPipeline.groovy [hunk 1]
```diff
@@ -17,95 +17,126 @@ def TEST_FLAG = (params.TEST_FLAG) ?: ""
...
-        }
-
-        def download_url = params.CUSTOMIZED_SDK_URL ? params.CUSTOMIZED_SDK_URL : ""
-        def sdk_resource_value = SDK_RESOURCE
-
-        if (SDK_RESOURCE == "customized" ) {
-            if (params.TOP_LEVEL_SDK_URL) {
-                // example: <jenkins_url>/job/build-scripts/job/openjdk11-pipeline/123/artifact/target/linux/aarch64/openj9/*_aarch64_linux_*.tar.gz/*zip*/openj9.zip 
-                download_url = params.TOP_LEVEL_SDK_URL + "artifact/target/${os}/${arch}/${params.VARIANT}/${filter}/*zip*/${params.VARIANT}.zip"
+node ("master") {
```
Comment (llxia) [inline buildenv/jenkins/aqaTestPipeline.groovy:20]: I think we cannot use `master`. If I remember it correctly, some Jenkins server does not have `master`.

Comment (sophia-guo) [reply buildenv/jenkins/aqaTestPipeline.groovy:20]: Yeah, forgot openj9 doesn't have master

**Comment by sophia-guo** [comment]
https://ci.adoptopenjdk.net/view/Test_grinder/job/AQA_Test_Pipeline/77/artifact/

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
👍

**Pull request merged**